src/adapters.ts

```diff
@@ -4,6 +4,7 @@ import moment from 'moment';
 import lru from 'lru-cache';
 import { bufToString, isBuf } from './buffer-node';
 import { compareVersions, nullIsh } from './utils';
+import Knex from 'knex';
```

Comment:
No third-party library must be imported explicitely in pg-mem, because:

- Most of them are not browser-compatible
- You usually only use one of them, thus, only have installed one of them... If you import 'knex' here, anyone using something else, say Typeorm, will start experiencing `module not found: knex` errors
- I dont want to add all supported libraries to pg-mem dependencies, because want to force anyone installing pg-mem to also install thousands of other modules they wont use.... that's why those are declared as optional `peerDependencies`

That is why no library is imported in Typescript... instead, I use `__non_webpack_require__`  INSIDE adapter calls, which webpack just compiles to `require()` .

The downside is: You cant have strongly typed arguments & return types... That's why everything returns `any` and takes `any`  as argument types on configs.

... so you'd have to do the same for this PR :/

Reply:
Ok, indeed. I'll adjust